sync_overview.adoc

```diff
@@ -128,6 +128,19 @@ That's it. You can now use the sync framework in your app. You can find the samp
 +
 However, if the default data access implementations don't meet your app's requirements, you can provide override functions.
 
+[[sync-loop-considerations]]
+=== Considerations to avoid unnecessary sync loops
+
+Two sync loops may be invoked per sync frequency if the server-side sync frequency differs from the client-side frequency.
+This is because the client and server sync frequencies are set independently. Setting a long frequency on a client does not change the sync frequency on the server.
+The syncFrequency value of the dataset on the server should be set to the sync_frequency value of the corresponding dataset on the client to avoid this.
+
+For example:
+* sync_frequency on the client-side dataset is also set to 120 seconds.
+* syncFrequency on the server-side dataset is set to 120 seconds.
```

Comment:
I think the `also` on L139 be on L140 instead? Or the two lines should be switched

```
sync_frequency on the client-side dataset is set to 120 seconds.
syncFrequency on the server-side dataset is also set to 120 seconds.
```